eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -58,12 +58,12 @@ function DeployStressTests(
     [hashtable]$filters = @{},
     [string]$environment = 'test',
     [string]$repository = '',
-    [boolean]$pushImages = $false,
+    [switch]$pushImages = $false,
```

Comment:
FWIW, defaulting to `$false` isn't necessary. Switches are `$false` by default. Doesn't hurt, but unnecessary.